Makefile

```
@@ -0,0 +1,57 @@
+.PHONY: help install test test-unit test-e2e test-e2e-headed lint format clean setup-dev
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)